client/src/components/areaDetail.tsx

```diff
@@ -2,91 +2,126 @@ import * as React from 'react';
-    for (let [key, value] of sortedKeys) {
-      // We should only format floats
-      if (typeof value === 'number' && value % 1 !== 0) {
-        value = readablePercent(value);
-      }
-
-      // Filter out all caps
-      if (!key.match(/^[A-Z0-9]+$/)) {
-        rows.push(<tr key={key} >
-          <td>{key}</td>
-          <td>{value}</td>
-        </tr>);
-      }
-    }
-    return rows;
+  const eduInfo:indicatorInfo = {
+    label: 'Education',
+    description: 'Percent of people age 25 or older that didn’t get a high school diploma',
+    value: properties[constants.EDUCATION_PROPERTY_PERCENTILE],
+  };
+  const linIsoInfo:indicatorInfo = {
+    label: 'Linguistic Isolation',
+    description: 'Households in which all members speak a non-English language and speak English less than "very well"',
+    value: properties[constants.LINGUISTIC_ISOLATION_PROPERTY_PERCENTILE],
+  };
+  const umemployInfo:indicatorInfo = {
+    label: 'Unemployment rate',
+    description: 'Number of unemployed people as a percentage of the labor force',
+    value: properties[constants.UNEMPLOYMENT_PROPERTY_PERCENTILE],
+  };
+  const houseBurden:indicatorInfo = {
+    label: 'Housing Burden',
+    description: 'Households that are low income and spend more than 30% of their income to housing costs',
+    value: properties[constants.HOUSING_BURDEN_PROPERTY_PERCENTILE],
   };
 
+  const indicators = [povertyInfo, eduInfo, linIsoInfo, umemployInfo, houseBurden];
+
+  const [categorization, categoryCircleStyle] = getCategorization(score);
 
   return (
-    <>
-      {properties ?
-      <div className={styles.areaDetailContainer}>
-        {getTitleContent()}
-        <div className={styles.areaDetailTableContainer}>
-          <table className={'usa-table usa-table--borderless ' + styles.areaDetailTable}>
-            <thead>
-              <tr>
-                <th scope="col">INDICATOR</th>
-                <th scope="col">VALUE</th>
-              </tr>
-            </thead>
-            <tbody>
-              {getBodyContent()}
-            </tbody>
-          </table>
+    <aside className={styles.areaDetailContainer}>
+      <header className={styles.topRow}>
+        <div className={styles.cumltveIndScore}>
+          <div className={styles.topRowTitle}>Cumulative Index Score</div>
+          <div className={styles.score}>{`${readablePercent(score)}`}
+            <sup className={styles.scoreSuperscript}><span>th</span></sup>
+          </div>
+          <div className={styles.topRowSubTitle}>percentile</div>
+        </div>
+        <div className={styles.categorization}>
+          <div className={styles.topRowTitle}>Categorization</div>
+          <div className={styles.priority}>
+            <div className={categoryCircleStyle} />
+            <div className={styles.prioritization}>{categorization}</div>
+          </div>
         </div>
-      </div> :
-    '' }
-    </>
+      </header>
+      <ul className={styles.censusRow}>
```

Comment:
Maybe log a ticket here for localizing this content? this may be a bigger discussion as we may want to have this on the backend; we'll likely get into this more as part of https://app.zenhub.com/workspaces/justice40-60993f6e05473d0010ec44e3/issues/usds/justice40-tool/367

Reply:
added intl here!